draft-amend-tsvwg-multipath-dccp.mkd

```diff
@@ -618,7 +662,12 @@ Smooth RTT (=3)
 {: vspace='0'}
 
 Age (=4)
-: [TBD]
+: The Age parameter is a 4-byte value which is set to the age or  
+  time stamp when the Ack for the datagram was received in case of  
```

Comment:
"timestamp" is used elsewhere in the doc

Reply:
I guess you proposed to remove the whitespace? Done